src/meta/src/hummock/manager/mod.rs

```diff
@@ -1290,8 +1299,20 @@ where
                 )?;
                 branched_ssts.commit_memory();
                 current_version.apply_version_delta(&version_delta);
+                let table_fragments: Vec<crate::model::TableFragments> =
+                    self.fragment_manager.list_table_fragments().await;
```

Comment:
list_table_fragments will clone lots of data. We must find a way to avoid it. We only need the mapping info of it.

Reply:
Got it, thank you for reminding.